addons/binding/org.openhab.binding.smaenergymeter/src/main/java/org/openhab/binding/smaenergymeter/handler/SMAEnergyMeterHandler.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2014 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.smaenergymeter.handler;
+
+import static org.openhab.binding.smaenergymeter.SMAEnergyMeterBindingConstants.*;
+
+import java.io.IOException;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.smaenergymeter.configuration.EnergyMeterConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link SMAEnergyMeterHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Osman Basha - Initial contribution
+ */
+public class SMAEnergyMeterHandler extends BaseThingHandler {
+
+    private Logger logger = LoggerFactory.getLogger(SMAEnergyMeterHandler.class);
+    private EnergyMeter energyMeter;
+    private ScheduledFuture<?> pollingJob;
+
+    public SMAEnergyMeterHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        logger.warn("This binding can not handle commands");
```

Comment:
It actually should support the "REFRESH" command, which triggers a refresh of the channel state (without having to wait for the polling)